tests/glm/test_utils.py

```diff
@@ -21,7 +21,7 @@ def df():
     )
 
 
-def test_align_df_categories_numeric(df):
+def testalign_df_categories_numeric(df):
```

Comment:
```suggestion
def test_align_df_categories_numeric(df):
```

And elsewhere. I imagine it's a typo.